src/js/tracks/text-track.js

```diff
@@ -226,11 +226,12 @@ class TextTrack extends Track {
             return;
           }
           mode = newMode;
-          if (mode === 'showing') {
-
+          if (mode !== 'disabled') {
             this.tech_.ready(() => {
               this.tech_.on('timeupdate', timeupdateHandler);
             }, true);
+          } else if (mode === 'disabled') {
```

Comment:
Wouldn't `} else {` make sense here?

Reply:
Potentially, let me see what browsers do for non-valid values.

Reply:
Chrome, Firefox, and Chrome will not let you set the value to anything that isn't hidden, showing, and disabled.

Reply:
Looks like we already ignore non-valid values, so, just else here is fine.